from botocore.exceptions import ClientError
import boto3

s3_client = boto3.client("s3")